src/components/App.tsx

```diff
@@ -2255,10 +2255,7 @@ class App extends React.Component<ExcalidrawProps, AppState> {
         touchTimeout = window.setTimeout(() => {
           touchTimeout = 0;
           if (!invalidateContextMenu) {
-            this.openContextMenu({
-              clientX: event.clientX,
-              clientY: event.clientY,
-            });
+            this.handleCanvasContextMenu(event);
```

Comment:
Since this now uses `handleCanvasContextMenu`, it prevent the event. I hope it's not an issue on mobile.